packages/jaeger-ui-components/src/TraceTimelineViewer/SpanTreeOffset.tsx

```diff
@@ -53,12 +53,9 @@ export const getStyles = createStyle((theme: Theme) => {
     `,
     indentGuideActive: css`
       label: indentGuideActive;
-      padding-right: calc(0.5rem + 11px);
-      border-left: 0px;
+      border-left: 3px solid darkgrey;
```

Comment:
Yup that looks identical to the upstream fix. But I can't help but wonder whether @everett980 is right about the fact that you only need to specify the color because `3px` and `solid` are already covered by `indentGuide`.

Reply:
Yep, that's correct. Using border-color is enough.